src/main/java/com/google/devtools/build/lib/bazel/repository/downloader/Checksum.java

```diff
@@ -64,41 +68,42 @@ private static Checksum fromString(KeyType keyType, String hash, boolean useSubr
         keyType, HashCode.fromString(Ascii.toLowerCase(hash)), useSubresourceIntegrity);
   }
 
+  private static byte[] base64Decode(String data) throws InvalidChecksumException {
+    try {
+      return Base64.getDecoder().decode(data);
+    } catch (IllegalArgumentException e) {
+      throw new InvalidChecksumException("Invalid base64 " + data, e);
```

Comment:
```suggestion
      throw new InvalidChecksumException("Invalid base64: " + data, e);
```

Clarifies where the invalid base64 starts as it might be *very* invalid.

Reply:
I will put the value in quotes since that's what the other error messages in this file do.